spec/models/trip_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+
+describe Trip do
```

Comment:
Will you also add assertions for the association (`belongs_to`) and the delegate method?

Reply:
I did it with fixup `85d5c3b`, not sure if it is enough, I will ask you tomorrow! 😸

Reply:
👍